### ISSUE TITLE
Can OBS build rpm for a customized Centos?

### ISSUE THREAD

**Issue by wufucious** [openSUSE/open-build-service#1010]
I have customized a Linux distribution from Centos 6.5. I hope OBS can build rpm packages for it. Is that possible? How?
Best regards,

**Comment by adrianschroeter** [comment]
Yes, OBS can do this most likely, but github issues are not for support tasks. Please go to the mailing list (or forum) to ask your questions.

**Comment by wufucious** [comment]
I have subscribe to opensuse-buildsservice@opensuse.org mailing list.
I got the confirmation letter.

Hi, this is the Mlmmj program managing the
<opensuse-buildservice@opensuse.org> mailing list.
 
Thank you for confirming your subscription. You have now been added to the
normal version of the list.
 
The email address you are subscribed with is <wufucious@139.com>.


But when I sent an email to opensuse-buildservice@opensuse.org, I still got this.
Hi, this is the Mlmmj program managing the
<opensuse-buildservice@opensuse.org> mailing list.
 
The message from <wufucious@139.com> with subject "[opensuse-buildservice]
Can OBS build rpm for a customized Centos?" was unable to be delivered to
the list because of an access rule set up by the list administrator.
 
Any suggestions?
Thanks.
@adrianschroeter 
How can I ask questions by mailing list?
